components/submit-grant/SubmitGrantFlow.tsx

```diff
@@ -0,0 +1,540 @@
+      inviteeNewOrganization: {
+        organization: {},
+      },
+      newPayoutMethod: {
+        data: {},
+      },
+    },
+    startOptions: startOptions.current,
+    handleOnSubmit: true,
+    onSuccess,
+    onError,
+  });
+
+  const steps = useExpenseFormSteps<Step>(
+    [
+      {
+        name: Step.INFORMATION,
+        title: <FormattedMessage defaultMessage="Information" id="E80WrK" />,
+        nextButtonMessage: <FormattedMessage defaultMessage="Proceed" id="VNX4fn" />,
+        items: [
+          {
+            name: Step.GRANT_PROVIDER,
+            hidden: true,
+            formValues: ['accountSlug'],
+          },
+          {
+            name: Step.INSTRUCTIONS,
+            hidden: true,
+            formValues: ['acknowledgedCollectiveGrantExpensePolicy', 'acknowledgedHostGrantExpensePolicy'],
+          },
+          {
+            name: Step.INVITATION_NOTE,
+            hidden: true,
+          },
+        ],
+      },
+      {
+        name: Step.APPLICATION_FORM,
+        title: <FormattedMessage defaultMessage="Application Form" id="TeGcrX" />,
+        previousButtonMessage: <FormattedMessage defaultMessage="Back" id="Back" />,
+        nextButtonMessage: <FormattedMessage defaultMessage="Proceed to Summary" id="tqKBWl" />,
+        items: [
+          {
+            name: Step.WHO_WILL_RECEIVE_FUNDS,
+            title: <FormattedMessage defaultMessage="Who will receive the fund?" id="q5ZkgW" />,
+            formValues: ['payeeSlug', 'inviteeNewIndividual', 'inviteeNewOrganization'],
+          },
+          {
+            name: Step.PAYOUT_METHOD,
+            title: <FormattedMessage defaultMessage="Payout Method" id="PayoutMethod" />,
+            formValues: ['payoutMethodId', 'newPayoutMethod', 'payoutMethodNameDiscrepancyReason'],
+          },
+          {
+            name: Step.DESCRIPTION,
+            title: <FormattedMessage defaultMessage="Description" id="Fields.description" />,
+            formValues: ['expenseItems'],
+          },
+        ],
+      },
+      {
+        name: Step.SUMMARY,
+        previousButtonMessage: <FormattedMessage defaultMessage="Edit Application" id="ByzFz7" />,
+        nextButtonMessage: <FormattedMessage defaultMessage="Submit Application" id="1vrJP6" />,
+        title: <FormattedMessage defaultMessage="Summary" id="Summary" />,
+      },
+    ],
+    expenseForm,
+    Step.INFORMATION,
+  );
+
+  const { setActiveStep } = steps;
+  const onVisibleSectionChange = React.useCallback(v => setActiveStep(v as Step), [setActiveStep]);
+
+  const { setFieldTouched, validateForm, handleSubmit } = expenseForm;
+
+  const onBackStepClick = React.useCallback(() => {
+    if (steps.activeHeaderName === Step.SUMMARY) {
+      setActiveStep(Step.APPLICATION_FORM);
+      return;
+    }
+
+    if (steps.activeHeaderName === Step.APPLICATION_FORM) {
+      setActiveStep(Step.INFORMATION);
+      return;
+    }
+
+    // TODO(henrique): exit form
```

Comment:
TODO here

Reply:
Not needed, all steps are covered by the ifs above and the back button is not displayed on 1s step.